src/components/UserMenu/ManaBalances/ManaBalances.tsx

```diff
@@ -1,29 +1,106 @@
 import React from "react"
 import { Network } from "@dcl/schemas"
+import { Tooltip } from "@mui/material"
 import { config } from "../../../config"
+import * as CreditsIcon from "../../Icon/CreditsIcon"
 import { Mana } from "../../Mana"
 import { ManaBalancesProps } from "./ManaBalances.types"
-import { ManaBalanceNumber, ManaBalancesWrapper } from "./ManaBalances.styled"
+import {
+  CreditsBalanceContainer,
+  CreditsBalanceTooltipContainer,
+  CreditsBalanceValueContainer,
+  ExperingSoon,
+  GetCreditsButton,
+  ManaBalanceNumber,
+  ManaBalancesWrapper,
+} from "./ManaBalances.styled"
 
 const ManaBalances = React.memo((props: ManaBalancesProps) => {
-  const { manaBalances, onClickBalance } = props
+  const { manaBalances, creditsBalance, i18n, onClickBalance } = props
+  console.log("i18n", i18n)
+
+  const isCreditsBalanceExpired =
+    creditsBalance && creditsBalance.expiresAt < Date.now()
+
+  const creditsExpiringSoon =
+    creditsBalance &&
+    creditsBalance.expiresAt < Date.now() + 1000 * 60 * 60 * 24 * 5 // 5 days
+
+  // calculates in how much time they expire. This returns days remaining
+  const creditsExpiringIn =
+    creditsBalance &&
+    Math.ceil((creditsBalance.expiresAt - Date.now()) / (1000 * 60 * 60 * 24))
 
   return (
     <ManaBalancesWrapper>
-      {manaBalances &&
-        Object.keys(manaBalances).map((network) => (
-          <Mana
-            key={network}
-            network={network as Network}
-            href={config.get("ACCOUNT_URL")}
-            onClick={onClickBalance}
-            primary={network === Network.ETHEREUM}
-          >
-            <ManaBalanceNumber variant="subtitle2">
-              {Number(manaBalances[network].toFixed(2)).toLocaleString()}
-            </ManaBalanceNumber>
-          </Mana>
-        ))}
+      <div>
+        {manaBalances &&
+          Object.keys(manaBalances).map((network) => (
+            <Mana
+              key={network}
+              network={network as Network}
+              href={config.get("ACCOUNT_URL")}
+              onClick={onClickBalance}
+              primary={network === Network.ETHEREUM}
+            >
+              <ManaBalanceNumber variant="subtitle2">
+                {Number(manaBalances[network].toFixed(2)).toLocaleString()}
+              </ManaBalanceNumber>
+            </Mana>
+          ))}
+      </div>
+      {creditsBalance !== undefined && !isCreditsBalanceExpired && (
+        <CreditsBalanceContainer>
+          <>
+            {creditsBalance.balance === 0 ? (
+              <>
+                <GetCreditsButton
+                  variant="text"
+                  onClick={() => {
+                    // TODO: review this link
+                    window.open(
+                      "https://docs.decentraland.org/decentraland/credits/",
```

Comment:
Would you mind adding this link into config envs?

Reply:
done! good idea